input/pagecontent/terminology.md

```diff
@@ -3,7 +3,7 @@ This page lists all ValueSets and CodeSystems defined as part of this specificat
 
 As well as these included definitions this specification typically references [National Clinical Terminology Service (NCTS)](https://www.healthterminologies.gov.au/integration/R4/fhir) terminology resources. 
 
-<p class="stu-note">We have introduced some terminology to support extensions around sex and gender. These extensions are currently under discussion with HL7 Australia and/or HL7 International. The terminology resources introduced are: Personal Pronouns value set and Recorded Sex Or Gender Type value set. These extensions, and the associated terminology, may be removed or changed prior to publication of the next version of this guide.</p>
+<p class="stu-note">We have introduced some terminology to support extensions around sex and gender. These extensions are currently under discussion with HL7 Australia and/or HL7 International. The terminology resources introduced are: Recorded Sex Or Gender Type value set. These extensions, and the associated terminology, may be removed or changed prior to publication of the next version of this guide.</p>
```

Comment:
A note may need to be present depending on outcome of review comments but this content is no longer relevant. 

None of this terminology is under discussion by either groups. None of the extensions are changing.

Question is why is this in the AU Base IG and not just a reference to the FHIR standard? Its technical.